packages/harpguru-core/src/components/toggle-buffer-flusher/hooks/index.ts

```diff
@@ -1 +1 @@
-export { useFlushBufferedActivityToggles } from './use-flush-buffered-activity-toggles'
+export { useRegularFlushBufferedToggles } from './use-regular-flush-buffered-activity-toggles'
```

Comment:
The name of the component and the hook should match. If this is to become a 'regular' toggle flusher then the component name could incorporate that. I think I'm going to need to come up with some kind of name like that anyway in order to define the enum for the global variable anyway.

Reply:
Nope. Changed my mind. I think it's more important that the component is named simply than totally consistently with it's hook.